unittests/inference/isinstance.py

```diff
@@ -0,0 +1,40 @@
+def isinstance(x: object, y: object) -> bool:
+    pass
+
+
+class A:
+    def f(self):
+        return 1
+
+
+class B:
+    def g(self):
+        return 1
+
+
+class C:
```

Comment:
I think this would be a reasonable example if you made A and B subclasses of C. Then f should get type C -> int. The way it is now, the program should not be typable.